packages/shared/src/atoms/SegmentedControl/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {
+  DetailedHTMLProps,
+  InputHTMLAttributes,
+  forwardRef,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+
+import * as S from './style'
+
+interface Props
+  extends DetailedHTMLProps<
+    InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  > {
+  text1: string
+  text2: string
+}
+
+const SegmentedControl = forwardRef<HTMLInputElement, Props>((props, ref) => {
```

Comment:
```suggestion
const SegmentedControl = forwardRef<HTMLInputElement, Props>(({ ...props }, ref) => {
```
sms에서는 구조분해 할당이 컨벤션에 있어서 되도록이면 지켜줘
물론 문서화가 안되서 문제긴 하지만...

Reply:
저기서 `{ text1, text2, ...props}` 이런식으로 하면 될거야

Reply:
b9919f669dfb7e59b07ca69881e7152fc97113ab
수정완료했습니다